tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -138,13 +138,13 @@
 
     <!-- Integration: NUnit3.TestAdapter -->
     <!-- Assembly: NUnit3.TestAdapter -->
-    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.1.0 -->
-    <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
+    <!-- Latest package https://www.nuget.org/packages/NUnit3TestAdapter/4.2.0 -->
+    <PackageReference Include="NUnit3TestAdapter" Version="4.2.0" />
```

Comment:
Looks like this package update isn't reflected in the package versions generated files

Reply:
Yeah, it doesn't, we don't actually test this adapter. IT's a very basic integration though, we hook the `Unload()` method to do a flush of the tracer. I've manually verified it all looks good.